src/backend/distributed/shared_library_init.c

```diff
@@ -720,6 +720,18 @@ RegisterCitusConfigVariables(void)
 		GUC_UNIT_KB,
 		NULL, NULL, NULL);
 
+	DefineCustomIntVariable(
+		"citus.max_pool_size",
```

Comment:
when we remove the old executors, what will be the meaning of 0?

Reply:
I guess the min value would be 1?